lib/kwtool/crawlers/upload_parser.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Kwtool.Crawlers.UploadParser do
+  alias NimbleCSV.RFC4180, as: CSV
+
+  def load_from_file(%Plug.Upload{content_type: "text/csv"} = keyword_file) do
```

Comment:
maybe `parse` is clearer and more accurate to what it is doing 🤔

Reply:
Updated! Thanks 👍